core/tests/test_validations.py

```diff
@@ -1,18 +1,18 @@
 from core import Actions, ContactPayload, Record, controller
-
+from core.misc.constants import InfoMessages, ValidationMessages
```

Comment:
could you please also apdate __init__.py in core.misc.constants so we can use shorter import path. In __init__.py should be example in this file how to do it.
But it could be done as followup, no need to do it in current PR

Reply:
Done